app/presenters/edition_presenter.rb

```diff
@@ -29,11 +29,11 @@ def initialize(edition, draft: false)
     end
 
     def for_content_store(payload_version)
-      present.except(:update_type).merge(payload_version: payload_version)
+      present.except(:update_type, :auth_bypass_ids).merge(payload_version: payload_version)
     end
 
     def for_message_queue(payload_version)
-      present.merge(
+      present.except(:auth_bypass_ids).merge(
```

Comment:
I'd suggest using the [NON_PRESENTED_PROPERTIES](https://github.com/alphagov/publishing-api/blob/b8c428c9039d1477bf558aea02a37a20b9e76281/app/presenters/edition_presenter.rb#L3-L24) constant for this rather than adding individual excepts.

Reply:
Sure, although this will be removed in favour of a singular except on the `for_message_queue` method at a later date when we're ready to talk to the content-store.